charts/galoy/templates/_helpers.tpl

```diff
@@ -1,16 +1,274 @@
-{{- define "galoy.jwtSecret" -}}
-{{- $secret := (lookup "v1" "Secret" .Release.Namespace "jwt-secret") -}}
-{{- if $secret -}}
+{{/* vim: set filetype=mustache: */}}
 {{/*
-   Reusing current password since secret exists
+Expand the name of the chart.
 */}}
-{{- $secret.data.secret -}}
-{{- else if .Values.jwtSecret -}}
-{{ .Values.jwtSecret | b64enc }}
-{{- else -}}
+{{- define "galoy.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified api name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "galoy.api.fullname" -}}
+{{- default "api" .Values.galoy.api.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified admin name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "galoy.admin.fullname" -}}
+{{- default "graphql-admin" .Values.galoy.admin.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified exporter name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "galoy.exporter.fullname" -}}
+{{- default "exporter" .Values.galoy.exporter.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified trigger name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "galoy.trigger.fullname" -}}
+{{- default "trigger" .Values.galoy.trigger.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+CronJob name
+*/}}
+{{- define "galoy.cron.jobname" -}}
+{{- printf "%s-cronjob" .Release.Name -}}
+{{- end -}}
+
+{{/*
+Balance Notification CronJob name
+*/}}
+{{- define "galoy.balance_notification_cron.jobname" -}}
+{{- printf "%s-balance-notif" .Release.Name -}}
+{{- end -}}
+
+{{/*
+Mongo Backup CronJob name
+*/}}
+{{- define "galoy.mongo_backup_cron.jobname" -}}
```

Comment:
Should probably stick to a case-convention for these keys. It's snake case here and kebab case on line 72 - `pre-migration`.
I've only seen camelcase being the convention in most helm charts, so we should think about this.